ivy_tests/test_ivy/test_functional/test_core/test_utility.py

```diff
@@ -1,3 +1,4 @@
+# for review
```

Comment:
we should not make any use of `data.draw`, we should either refactor into private helper functions in this test file itself, or add the helper functions to the `helpers.py` file for other tests to make use of.